src/utilities/findBreakingChanges.js

```diff
@@ -363,3 +545,36 @@ export function findValuesRemovedFromEnums(
   });
   return valuesRemovedFromEnums;
 }
+
+export function findInterfacesRemovedFromObjectTypes(
+  oldSchema: GraphQLSchema,
+  newSchema: GraphQLSchema
+): Array<BreakingChange> {
+  const oldTypeMap = oldSchema.getTypeMap();
+  const newTypeMap = newSchema.getTypeMap();
+  const breakingChanges = [];
+
+  Object.keys(oldTypeMap).forEach(typeName => {
+    const oldType = oldTypeMap[typeName];
+    const newType = newTypeMap[typeName];
+    if (
+      !(oldType instanceof GraphQLObjectType) ||
+      !(newType instanceof oldType.constructor)
```

Comment:
Yeah this pattern makes more sense in some of the other functions like in `findFieldsThatChangedTypeOnObjectOrInterfaceTypes` where we have:
```
if (
  !(oldType instanceof GraphQLObjectType ||
  oldType instanceof GraphQLInterfaceType) ||
  !(newType instanceof oldType.constructor)
) {
  return;
}
...
```

We want to make sure that `oldType` is either a `GraphQLObjectType` or a `GraphQLInterfaceType` and that the `newType` is the same kind. (If they are different kinds than that is a more major breaking change that is caught by `findTypesThatChangedKind`.

In this case though, it does look odd when we're only checking for one particular kind of type. I changed it so that we also check that `!(newType instanceof GraphQLObjectType)`. (I also changed other places where there's only a single kind of type we want).